content/en/security/application_security/enabling/single_step/code_security.md

```diff
@@ -0,0 +1,420 @@
+```
+
+You can optionally provide a version number for the tracing library by placing a colon after the language name and specifying the tracing library version. If you don't specify a version, it defaults to the latest version. Language names are comma-separated.
+
+Supported languages include:
+
+- .NET (`dotnet`)
+- Python (`python`)
+- Java (`java`)
+- Node.js (`js`)
+- Ruby (`ruby`)
+
+**Note**: For the Node.js tracing library, different versions of Node.js are compatible with different versions of the Node.js tracing library. See [DataDog/dd-trace-js: JavaScript APM Tracer][7] for more information.
+
+### Tagging observability data by environment {#env-docker}
+
+Set `DD_ENV` in the library injector installation command for Docker to automatically tag instrumented services and other telemetry that pass through the Agent with a specific environment. For example, if the Agent is installed in your staging environment, set `DD_ENV=staging` to associate your observability data with `staging`.
+
+For example:
+
+{{< highlight shell "hl_lines=5" >}}
+docker run -d --name dd-agent \
+  -e DD_API_KEY=${YOUR_DD_API_KEY} \
+  -e DD_APM_ENABLED=true \
+  -e DD_IAST_ENABLED=true \ 
+  -e DD_ENV=staging \
+  -e DD_APM_NON_LOCAL_TRAFFIC=true \
+  -e DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true \
+  -e DD_APM_RECEIVER_SOCKET=/opt/datadog/apm/inject/run/apm.socket \
+  -e DD_DOGSTATSD_SOCKET=/opt/datadog/apm/inject/run/dsd.socket \
+  -v /opt/datadog/apm:/opt/datadog/apm \
+  -v /var/run/docker.sock:/var/run/docker.sock:ro \
+  gcr.io/datadoghq/agent:7
+{{< /highlight >}}
+
+[5]: https://app.datadoghq.com/organization-settings/api-keys
+[6]: /service_catalog/
+[7]: https://github.com/DataDog/dd-trace-js?tab=readme-ov-file#version-release-lines-and-maintenance
+
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+You can enable APM by installing the Agent with the Datadog Helm chart. This deploys the Datadog Agent across all nodes in your Linux-based Kubernetes cluster with a DaemonSet.
+
+**Note**: Single step instrumentation doesn't instrument applications in the namespace where you install the Datadog Agent. It's recommended to install the Agent in a separate namespace in your cluster where you don't run your applications.
+
+### Requirements
+
+- Make sure you have [Helm][13] installed.
+
+### Installation
+
+To enable single step instrumentation with Helm:
+
+1. Add the Helm Datadog repo:
+   ```bash
+    helm repo add datadog https://helm.datadoghq.com
+    helm repo update
+    ```
+2. Create a Kubernetes secret to store your Datadog [API key][10]:
+   ```bash
+   kubectl create secret generic datadog-secret --from-literal api-key=$DD_API_KEY
+   ```
+3. Create `datadog-values.yaml` and add the following configuration:
+   ```
+   datadog:
+    apiKeyExistingSecret: datadog-secret
+    site: <DATADOG_SITE>
+    apm:
+      instrumentation:
+         enabled: true
+    env:
+      - name: DD_IAST_ENABLED=true 
+        value: "true"
```

Comment:
I don't think this part is going to work as is.

Looking at the docs tells me that `datadog.env` field adds this environment variable on all *agent* pods but this environment variable should be sent to all *traced* pods for this to work.

I think what you are looking for is:
```suggestion
    asm:
      iast:
        enabled: true
```

Cf. The [PR](https://github.com/DataDog/helm-charts/pull/1337/files) I made in the helm charts that should be merged once the new version of the cluster agent is released.